server/views/curfew/approvedPremisesAddress.pug

```diff
@@ -1,5 +1,4 @@
 extends ../layout
-include ../proposedAddress/includes/addressDetail
```

Comment:
Just a check but definitely don't need this?

Reply:
I'm pretty sure as I renamed addressDetail to either addressDetailV1 or V2. Checked it wasn't used in other places too.

Reply:
And so do we not want the address details displaying here anymore?

Reply:
The addressDetail component isn't currently being rendered in the application at the moment. Although it's been 'included' on line 2 of approvedPremisesAddress.pg it's not being called elsewhere hence I removed it

Reply:
Great will approve